tests/test_import.py

```diff
@@ -156,7 +155,7 @@ def test_thorough_ndex(self):
     def test_thorough_upgrade(self):
         lines = to_bel_lines(self.thorough_graph)
         reconstituted = from_lines(lines, manager=self.manager)
-        self.bel_thorough_reconstituted(reconstituted)
+        self.bel_thorough_reconstituted(reconstituted, check_warnings=False)
```

Comment:
Remove `check_warnings=False`. I believe I have already addressed the underlying problem!